tsconfig.json

```diff
@@ -5,6 +5,7 @@
     "allowJs": true,
     "noEmit": true,
     "noImplicitAny": false,
+    "jsx": "react",
```

Comment:
This keys in IDEs that jsx/tsx files depend on react, so they don't try to remove the import as unused.